generate-stackbrew-library.sh

```diff
@@ -55,22 +55,26 @@ get_stub() {
 
 for version in "${versions[@]}"; do
 	# Skip "docs" and other non-docker directories
-	[ -f "$version/Dockerfile" ] || continue
+	[ -f "$version/Dockerfile" ] || [ -f "$version/$default_variant/Dockerfile" ] || continue
```

Comment:
Is `[ -f "$version/Dockerfile" ]` still required after moving the default `Dockerfile`?

Reply:
It's required for Chakracore